ui/de-webapp/src/main/java/org/iplantc/de/server/websocket/MessageHandler.java

```diff
@@ -65,7 +65,7 @@ public void onDisconnect(AtmosphereResourceEvent event) {
                     }
                 } catch (IOException e) {
                     e.printStackTrace();
-                    logger.error("^^^^^^^^^^^ exception aborting channel! ^^^^^^^^");
+                    logger.error("Exception aborting channel:" + e.getMessage());
```

Comment:
Is it possible to pass the Throwable as another param to these `error` log messages, rather than concatenating the exception message and separately calling the `printStackTrace` method?

For example:
```java
logger.error("Exception aborting channel", e);
```